Exercises-1-Basic/src/exercises/Num3.java

```diff
@@ -0,0 +1,25 @@
+package exercises;
+
+import java.util.Scanner;
+
+/*
+ * Write a program that prompts the user to input a positive integer.
+ *  It should then print the multiplication table of that number. 
+ */
+public class Num3 {
+    public static void main(String[] args) {
+
+	Scanner sc = new Scanner(System.in);
+
+	System.out.println("Your number: ");
+	int Yournumber = sc.nextInt();
```

Comment:
```suggestion
	int yourNumber = sc.nextInt();
```
Variable names should be in camleCase format.

Reply:
Done.